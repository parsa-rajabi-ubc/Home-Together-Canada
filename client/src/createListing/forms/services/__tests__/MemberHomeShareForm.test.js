/**
 * @Author:     Parsa Rajabi
 * @Created:    2021.2.4
 *
 * @Description: Member Home Share Form Test
 *
 */

import React from 'react';
import renderer from  'react-test-renderer'
import MemberHomeShareForm from "../MemberHomeShareForm";

jest.mock("react-tooltip/node_modules/uuid", () => ({
            v4: () => "00000000-0000-0000-0000-000000000000"}
    )
);

describe('MemberHomeShareForm', () => {
    describe('Snapshot test', () => {
        it("should render correctly regardless of properties", () => {
            // given
            const props = {
                onSubmit: jest.fn()
            };
            //when
            const component = renderer.create(<MemberHomeShareForm {...props}/>).toJSON();
            //then
            expect(component).toMatchSnapshot();
        });
    })
})