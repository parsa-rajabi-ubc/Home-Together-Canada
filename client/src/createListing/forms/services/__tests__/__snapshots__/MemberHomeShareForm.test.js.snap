// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MemberHomeShareForm Snapshot test should render correctly regardless of properties 1`] = `
<div>
  <div
    className="col-start-1 col-end-7 py-5 px-5 m-6 bg-white shadow-lg rounded-xl"
  >
    <div
      className="grid grid-cols-2 gap-6"
    >
      <div
        className="col-span-3 sm:col-span-2"
      >
        <h1
          className="page-title mb-5"
        >
           
          Members with Homes to Share
           
        </h1>
        <section>
          <label
            className="label"
          >
            Descriptive Title
          </label>
          <svg
            className="inline align-top text-red-600"
            fill="none"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M11 6H13V10.079L16.3413 7.73938L17.4885 9.37768L13.7434 12L17.4885 14.6223L16.3413 16.2606L13 13.921V18H11V13.921L7.65864 16.2606L6.51148 14.6223L10.2565 12L6.51147 9.37769L7.65863 7.73938L11 10.079V6Z"
              fill="currentColor"
            />
          </svg>
          <input
            className="false undefined input"
            disabled={false}
            maxLength={75}
            onChange={[Function]}
            type="text"
            value=""
          />
        </section>
        <div
          className="grid grid-cols-9"
        >
          <section
            className="col-start-1 col-end-5"
          >
            <section>
              <label
                className="label"
              >
                Short Description (50 Characters)
              </label>
              <svg
                className="inline align-top text-red-600"
                fill="none"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M11 6H13V10.079L16.3413 7.73938L17.4885 9.37768L13.7434 12L17.4885 14.6223L16.3413 16.2606L13 13.921V18H11V13.921L7.65864 16.2606L6.51148 14.6223L10.2565 12L6.51147 9.37769L7.65863 7.73938L11 10.079V6Z"
                  fill="currentColor"
                />
              </svg>
              <input
                className="false undefined input"
                disabled={false}
                maxLength={50}
                onChange={[Function]}
                type="text"
                value=""
              />
            </section>
          </section>
          <section
            className="undefined my-2 col-start-6 col-end-9"
          >
            <div>
              <div>
                <label
                  className="label text-base"
                >
                  Utilities included?
                </label>
                <svg
                  className="inline align-top text-red-600"
                  fill="none"
                  height="1em"
                  stroke="currentColor"
                  strokeWidth="0"
                  style={
                    Object {
                      "color": undefined,
                    }
                  }
                  viewBox="0 0 24 24"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M11 6H13V10.079L16.3413 7.73938L17.4885 9.37768L13.7434 12L17.4885 14.6223L16.3413 16.2606L13 13.921V18H11V13.921L7.65864 16.2606L6.51148 14.6223L10.2565 12L6.51147 9.37769L7.65863 7.73938L11 10.079V6Z"
                    fill="currentColor"
                  />
                </svg>
              </div>
              <span
                className="false"
              >
                <label
                  className="text-base text-gray-700 pr-8"
                >
                  <input
                    checked={false}
                    className="form-radio mt-2 ml-2 mr-1 pb-4"
                    name="utilIncluded"
                    onChange={[Function]}
                    type="radio"
                    value="no"
                  />
                  No
                </label>
              </span>
              <span
                className="false"
              >
                <label
                  className="text-base text-gray-700 pr-8"
                >
                  <input
                    checked={false}
                    className="form-radio mt-2 ml-2 mr-1 pb-4"
                    name="utilIncluded"
                    onChange={[Function]}
                    type="radio"
                    value="yes"
                  />
                  Yes
                </label>
              </span>
            </div>
          </section>
          <section
            className="col-start-1 col-end-5"
          >
            <label
              className="label"
            >
              Home share monthly cost (CAD)
               
              <svg
                className="inline align-top text-red-600"
                fill="none"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M11 6H13V10.079L16.3413 7.73938L17.4885 9.37768L13.7434 12L17.4885 14.6223L16.3413 16.2606L13 13.921V18H11V13.921L7.65864 16.2606L6.51148 14.6223L10.2565 12L6.51147 9.37769L7.65863 7.73938L11 10.079V6Z"
                  fill="currentColor"
                />
              </svg>
            </label>
            <input
              className="undefined input"
              min="0"
              onChange={[Function]}
              step="1"
              type="number"
            />
          </section>
          <section
            className="undefined my-2 col-start-6 col-end-9"
          >
            <div>
              <div>
                <label
                  className="label text-base"
                >
                  Pet friendly?
                </label>
                <svg
                  className="inline align-top text-red-600"
                  fill="none"
                  height="1em"
                  stroke="currentColor"
                  strokeWidth="0"
                  style={
                    Object {
                      "color": undefined,
                    }
                  }
                  viewBox="0 0 24 24"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M11 6H13V10.079L16.3413 7.73938L17.4885 9.37768L13.7434 12L17.4885 14.6223L16.3413 16.2606L13 13.921V18H11V13.921L7.65864 16.2606L6.51148 14.6223L10.2565 12L6.51147 9.37769L7.65863 7.73938L11 10.079V6Z"
                    fill="currentColor"
                  />
                </svg>
              </div>
              <span
                className="false"
              >
                <label
                  className="text-base text-gray-700 pr-8"
                >
                  <input
                    checked={false}
                    className="form-radio mt-2 ml-2 mr-1 pb-4"
                    name="petFriendly"
                    onChange={[Function]}
                    type="radio"
                    value="no"
                  />
                  No
                </label>
              </span>
              <span
                className="false"
              >
                <label
                  className="text-base text-gray-700 pr-8"
                >
                  <input
                    checked={false}
                    className="form-radio mt-2 ml-2 mr-1 pb-4"
                    name="petFriendly"
                    onChange={[Function]}
                    type="radio"
                    value="yes"
                  />
                  Yes
                </label>
              </span>
            </div>
          </section>
          <section
            className="col-start-1 col-end-5"
          >
            <label
              className="label"
            >
              Number of bedrooms
               
              <svg
                className="inline align-top text-red-600"
                fill="none"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M11 6H13V10.079L16.3413 7.73938L17.4885 9.37768L13.7434 12L17.4885 14.6223L16.3413 16.2606L13 13.921V18H11V13.921L7.65864 16.2606L6.51148 14.6223L10.2565 12L6.51147 9.37769L7.65863 7.73938L11 10.079V6Z"
                  fill="currentColor"
                />
              </svg>
            </label>
            <div>
              <div
                className=" css-2b097c-container"
                onKeyDown={[Function]}
              >
                <div
                  className=" css-1jumw1e-control"
                  onMouseDown={[Function]}
                  onTouchEnd={[Function]}
                >
                  <div
                    className=" css-g1d714-ValueContainer"
                  >
                    <div
                      className=" css-1d6d5xl-placeholder"
                    >
                      Select...
                    </div>
                    <div
                      className="css-b8ldur-Input"
                    >
                      <div
                        className=""
                        style={
                          Object {
                            "display": "inline-block",
                          }
                        }
                      >
                        <input
                          aria-autocomplete="list"
                          autoCapitalize="none"
                          autoComplete="off"
                          autoCorrect="off"
                          disabled={false}
                          id="react-select-2-input"
                          onBlur={[Function]}
                          onChange={[Function]}
                          onFocus={[Function]}
                          spellCheck="false"
                          style={
                            Object {
                              "background": 0,
                              "border": 0,
                              "boxSizing": "content-box",
                              "color": "inherit",
                              "fontSize": "inherit",
                              "label": "input",
                              "opacity": 1,
                              "outline": 0,
                              "padding": 0,
                              "width": "1px",
                            }
                          }
                          tabIndex="0"
                          type="text"
                          value=""
                        />
                        <div
                          style={
                            Object {
                              "height": 0,
                              "left": 0,
                              "overflow": "scroll",
                              "position": "absolute",
                              "top": 0,
                              "visibility": "hidden",
                              "whiteSpace": "pre",
                            }
                          }
                        >
                          
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    className=" css-1hb7zxy-IndicatorsContainer"
                  >
                    <span
                      className=" css-1okebmr-indicatorSeparator"
                    />
                    <div
                      aria-hidden="true"
                      className=" css-tlfecz-indicatorContainer"
                      onMouseDown={[Function]}
                      onTouchEnd={[Function]}
                    >
                      <svg
                        aria-hidden="true"
                        className="css-6q0nyr-Svg"
                        focusable="false"
                        height={20}
                        viewBox="0 0 20 20"
                        width={20}
                      >
                        <path
                          d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                        />
                      </svg>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </section>
          <section
            className="undefined my-2 col-start-6 col-end-9"
          >
            <div>
              <div>
                <label
                  className="label text-base"
                >
                  Smoke friendly?
                </label>
                <svg
                  className="inline align-top text-red-600"
                  fill="none"
                  height="1em"
                  stroke="currentColor"
                  strokeWidth="0"
                  style={
                    Object {
                      "color": undefined,
                    }
                  }
                  viewBox="0 0 24 24"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M11 6H13V10.079L16.3413 7.73938L17.4885 9.37768L13.7434 12L17.4885 14.6223L16.3413 16.2606L13 13.921V18H11V13.921L7.65864 16.2606L6.51148 14.6223L10.2565 12L6.51147 9.37769L7.65863 7.73938L11 10.079V6Z"
                    fill="currentColor"
                  />
                </svg>
              </div>
              <span
                className="false"
              >
                <label
                  className="text-base text-gray-700 pr-8"
                >
                  <input
                    checked={false}
                    className="form-radio mt-2 ml-2 mr-1 pb-4"
                    name="smokeFriendly"
                    onChange={[Function]}
                    type="radio"
                    value="no"
                  />
                  No
                </label>
              </span>
              <span
                className="false"
              >
                <label
                  className="text-base text-gray-700 pr-8"
                >
                  <input
                    checked={false}
                    className="form-radio mt-2 ml-2 mr-1 pb-4"
                    name="smokeFriendly"
                    onChange={[Function]}
                    type="radio"
                    value="yes"
                  />
                  Yes
                </label>
              </span>
            </div>
          </section>
          <section
            className="col-start-1 col-end-5"
          >
            <label
              className="label"
            >
              Number of bathrooms
               
              <svg
                className="inline align-top text-red-600"
                fill="none"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M11 6H13V10.079L16.3413 7.73938L17.4885 9.37768L13.7434 12L17.4885 14.6223L16.3413 16.2606L13 13.921V18H11V13.921L7.65864 16.2606L6.51148 14.6223L10.2565 12L6.51147 9.37769L7.65863 7.73938L11 10.079V6Z"
                  fill="currentColor"
                />
              </svg>
            </label>
            <div>
              <div
                className=" css-2b097c-container"
                onKeyDown={[Function]}
              >
                <div
                  className=" css-1jumw1e-control"
                  onMouseDown={[Function]}
                  onTouchEnd={[Function]}
                >
                  <div
                    className=" css-g1d714-ValueContainer"
                  >
                    <div
                      className=" css-1d6d5xl-placeholder"
                    >
                      Select...
                    </div>
                    <div
                      className="css-b8ldur-Input"
                    >
                      <div
                        className=""
                        style={
                          Object {
                            "display": "inline-block",
                          }
                        }
                      >
                        <input
                          aria-autocomplete="list"
                          autoCapitalize="none"
                          autoComplete="off"
                          autoCorrect="off"
                          disabled={false}
                          id="react-select-3-input"
                          onBlur={[Function]}
                          onChange={[Function]}
                          onFocus={[Function]}
                          spellCheck="false"
                          style={
                            Object {
                              "background": 0,
                              "border": 0,
                              "boxSizing": "content-box",
                              "color": "inherit",
                              "fontSize": "inherit",
                              "label": "input",
                              "opacity": 1,
                              "outline": 0,
                              "padding": 0,
                              "width": "1px",
                            }
                          }
                          tabIndex="0"
                          type="text"
                          value=""
                        />
                        <div
                          style={
                            Object {
                              "height": 0,
                              "left": 0,
                              "overflow": "scroll",
                              "position": "absolute",
                              "top": 0,
                              "visibility": "hidden",
                              "whiteSpace": "pre",
                            }
                          }
                        >
                          
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    className=" css-1hb7zxy-IndicatorsContainer"
                  >
                    <span
                      className=" css-1okebmr-indicatorSeparator"
                    />
                    <div
                      aria-hidden="true"
                      className=" css-tlfecz-indicatorContainer"
                      onMouseDown={[Function]}
                      onTouchEnd={[Function]}
                    >
                      <svg
                        aria-hidden="true"
                        className="css-6q0nyr-Svg"
                        focusable="false"
                        height={20}
                        viewBox="0 0 20 20"
                        width={20}
                      >
                        <path
                          d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                        />
                      </svg>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </section>
          <section
            className="col-start-1 col-end-6"
          >
            <div
              className="grid grid-cols-9 gap-x-2"
            >
              <div
                className="col-start-1 col-end-8"
              >
                <label
                  className="label"
                >
                  General Location
                </label>
                <svg
                  className="inline align-top text-red-600"
                  fill="none"
                  height="1em"
                  stroke="currentColor"
                  strokeWidth="0"
                  style={
                    Object {
                      "color": undefined,
                    }
                  }
                  viewBox="0 0 24 24"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M11 6H13V10.079L16.3413 7.73938L17.4885 9.37768L13.7434 12L17.4885 14.6223L16.3413 16.2606L13 13.921V18H11V13.921L7.65864 16.2606L6.51148 14.6223L10.2565 12L6.51147 9.37769L7.65863 7.73938L11 10.079V6Z"
                    fill="currentColor"
                  />
                </svg>
                <span>
                  <span
                    data-for="memberHomeToShareAddress"
                    data-tip={true}
                  >
                    <svg
                      className="inline ml-1 mb-1 text-blue-500 hover:text-blue-400"
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 16 16"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        clipRule="evenodd"
                        d="M2 0a2 2 0 00-2 2v12a2 2 0 002 2h12a2 2 0 002-2V2a2 2 0 00-2-2H2zm4.57 6.033H5.25C5.22 4.147 6.68 3.5 8.006 3.5c1.397 0 2.673.73 2.673 2.24 0 1.08-.635 1.594-1.244 2.057-.737.559-1.01.768-1.01 1.486v.355H7.117l-.007-.463c-.038-.927.495-1.498 1.168-1.987.59-.444.965-.736.965-1.371 0-.825-.628-1.168-1.314-1.168-.901 0-1.358.603-1.358 1.384zm1.251 6.443c-.584 0-1.009-.394-1.009-.927 0-.552.425-.94 1.01-.94.609 0 1.028.388 1.028.94 0 .533-.42.927-1.029.927z"
                        fillRule="evenodd"
                      />
                    </svg>
                  </span>
                  <div
                    className="__react_component_tooltip t00000000-0000-0000-0000-000000000000 place-top type-dark"
                    data-id="tooltip"
                    id="memberHomeToShareAddress"
                  >
                    <style
                      dangerouslySetInnerHTML={
                        Object {
                          "__html": "
  	.t00000000-0000-0000-0000-000000000000 {
	    color: #fff;
	    background: #222;
	    border: 1px solid transparent;
  	}

  	.t00000000-0000-0000-0000-000000000000.place-top {
        margin-top: -10px;
    }
    .t00000000-0000-0000-0000-000000000000.place-top::before {
        border-top: 8px solid transparent;
    }
    .t00000000-0000-0000-0000-000000000000.place-top::after {
        border-left: 8px solid transparent;
        border-right: 8px solid transparent;
        bottom: -6px;
        left: 50%;
        margin-left: -8px;
        border-top-color: #222;
        border-top-style: solid;
        border-top-width: 6px;
    }

    .t00000000-0000-0000-0000-000000000000.place-bottom {
        margin-top: 10px;
    }
    .t00000000-0000-0000-0000-000000000000.place-bottom::before {
        border-bottom: 8px solid transparent;
    }
    .t00000000-0000-0000-0000-000000000000.place-bottom::after {
        border-left: 8px solid transparent;
        border-right: 8px solid transparent;
        top: -6px;
        left: 50%;
        margin-left: -8px;
        border-bottom-color: #222;
        border-bottom-style: solid;
        border-bottom-width: 6px;
    }

    .t00000000-0000-0000-0000-000000000000.place-left {
        margin-left: -10px;
    }
    .t00000000-0000-0000-0000-000000000000.place-left::before {
        border-left: 8px solid transparent;
    }
    .t00000000-0000-0000-0000-000000000000.place-left::after {
        border-top: 5px solid transparent;
        border-bottom: 5px solid transparent;
        right: -6px;
        top: 50%;
        margin-top: -4px;
        border-left-color: #222;
        border-left-style: solid;
        border-left-width: 6px;
    }

    .t00000000-0000-0000-0000-000000000000.place-right {
        margin-left: 10px;
    }
    .t00000000-0000-0000-0000-000000000000.place-right::before {
        border-right: 8px solid transparent;
    }
    .t00000000-0000-0000-0000-000000000000.place-right::after {
        border-top: 5px solid transparent;
        border-bottom: 5px solid transparent;
        left: -6px;
        top: 50%;
        margin-top: -4px;
        border-right-color: #222;
        border-right-style: solid;
        border-right-width: 6px;
    }
  ",
                        }
                      }
                    />
                    Please enter the general location of the house
                  </div>
                </span>
                <input
                  autoComplete="address-line1"
                  className="undefined input"
                  maxLength={100}
                  name="street"
                  onChange={[Function]}
                  placeholder="Address Line 1"
                  type="text"
                  value=""
                />
              </div>
              <div
                className="col-start-1 col-end-8"
              >
                <input
                  autoComplete="address-line2"
                  className="input"
                  maxLength={100}
                  name="aptNum"
                  onChange={[Function]}
                  placeholder="Address Line 2: Apt, suite, etc. (optional)"
                  type="text"
                  value=""
                />
              </div>
              <div
                className="col-start-1 col-end-4"
              >
                <input
                  autoComplete="address-level2"
                  className="undefined input"
                  maxLength={60}
                  name="city"
                  onChange={[Function]}
                  placeholder="City"
                  type="text"
                  value=""
                />
              </div>
              <div
                className="col-start-4 col-end-8"
              >
                <div>
                  <div
                    className=" css-2b097c-container"
                    onKeyDown={[Function]}
                  >
                    <div
                      className=" css-1jumw1e-control"
                      onMouseDown={[Function]}
                      onTouchEnd={[Function]}
                    >
                      <div
                        className=" css-g1d714-ValueContainer"
                      >
                        <div
                          className=" css-1d6d5xl-placeholder"
                        >
                          Province
                        </div>
                        <div
                          className="css-b8ldur-Input"
                        >
                          <div
                            className=""
                            style={
                              Object {
                                "display": "inline-block",
                              }
                            }
                          >
                            <input
                              aria-autocomplete="list"
                              autoCapitalize="none"
                              autoComplete="off"
                              autoCorrect="off"
                              disabled={false}
                              id="react-select-4-input"
                              onBlur={[Function]}
                              onChange={[Function]}
                              onFocus={[Function]}
                              spellCheck="false"
                              style={
                                Object {
                                  "background": 0,
                                  "border": 0,
                                  "boxSizing": "content-box",
                                  "color": "inherit",
                                  "fontSize": "inherit",
                                  "label": "input",
                                  "opacity": 1,
                                  "outline": 0,
                                  "padding": 0,
                                  "width": "1px",
                                }
                              }
                              tabIndex="0"
                              type="text"
                              value=""
                            />
                            <div
                              style={
                                Object {
                                  "height": 0,
                                  "left": 0,
                                  "overflow": "scroll",
                                  "position": "absolute",
                                  "top": 0,
                                  "visibility": "hidden",
                                  "whiteSpace": "pre",
                                }
                              }
                            >
                              
                            </div>
                          </div>
                        </div>
                      </div>
                      <div
                        className=" css-1hb7zxy-IndicatorsContainer"
                      >
                        <span
                          className=" css-1okebmr-indicatorSeparator"
                        />
                        <div
                          aria-hidden="true"
                          className=" css-tlfecz-indicatorContainer"
                          onMouseDown={[Function]}
                          onTouchEnd={[Function]}
                        >
                          <svg
                            aria-hidden="true"
                            className="css-6q0nyr-Svg"
                            focusable="false"
                            height={20}
                            viewBox="0 0 20 20"
                            width={20}
                          >
                            <path
                              d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                            />
                          </svg>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                className="col-start-1 col-end-4"
              >
                <input
                  autoComplete="postal-code"
                  className="undefined input"
                  maxLength={6}
                  name="postalCode"
                  onChange={[Function]}
                  placeholder="Postal Code"
                  type="text"
                  value=""
                />
              </div>
            </div>
          </section>
        </div>
        <div>
          <label
            className="label"
          >
            Full Description
          </label>
          <svg
            className="inline align-top text-red-600"
            fill="none"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M11 6H13V10.079L16.3413 7.73938L17.4885 9.37768L13.7434 12L17.4885 14.6223L16.3413 16.2606L13 13.921V18H11V13.921L7.65864 16.2606L6.51148 14.6223L10.2565 12L6.51147 9.37769L7.65863 7.73938L11 10.079V6Z"
              fill="currentColor"
            />
          </svg>
          <textarea
            className="undefined undefined undefined input"
            cols="50"
            maxLength={2000}
            onChange={[Function]}
            rows="6"
          />
        </div>
        <div>
          <label
            className="label"
          >
             Photos 
          </label>
          <span>
            <span
              data-for="UploadPhotos"
              data-tip={true}
            >
              <svg
                className="inline ml-1 mb-1 text-blue-500 hover:text-blue-400"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 16 16"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clipRule="evenodd"
                  d="M2 0a2 2 0 00-2 2v12a2 2 0 002 2h12a2 2 0 002-2V2a2 2 0 00-2-2H2zm4.57 6.033H5.25C5.22 4.147 6.68 3.5 8.006 3.5c1.397 0 2.673.73 2.673 2.24 0 1.08-.635 1.594-1.244 2.057-.737.559-1.01.768-1.01 1.486v.355H7.117l-.007-.463c-.038-.927.495-1.498 1.168-1.987.59-.444.965-.736.965-1.371 0-.825-.628-1.168-1.314-1.168-.901 0-1.358.603-1.358 1.384zm1.251 6.443c-.584 0-1.009-.394-1.009-.927 0-.552.425-.94 1.01-.94.609 0 1.028.388 1.028.94 0 .533-.42.927-1.029.927z"
                  fillRule="evenodd"
                />
              </svg>
            </span>
            <div
              className="__react_component_tooltip t00000000-0000-0000-0000-000000000000 place-top type-dark"
              data-id="tooltip"
              id="UploadPhotos"
            >
              <style
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "
  	.t00000000-0000-0000-0000-000000000000 {
	    color: #fff;
	    background: #222;
	    border: 1px solid transparent;
  	}

  	.t00000000-0000-0000-0000-000000000000.place-top {
        margin-top: -10px;
    }
    .t00000000-0000-0000-0000-000000000000.place-top::before {
        border-top: 8px solid transparent;
    }
    .t00000000-0000-0000-0000-000000000000.place-top::after {
        border-left: 8px solid transparent;
        border-right: 8px solid transparent;
        bottom: -6px;
        left: 50%;
        margin-left: -8px;
        border-top-color: #222;
        border-top-style: solid;
        border-top-width: 6px;
    }

    .t00000000-0000-0000-0000-000000000000.place-bottom {
        margin-top: 10px;
    }
    .t00000000-0000-0000-0000-000000000000.place-bottom::before {
        border-bottom: 8px solid transparent;
    }
    .t00000000-0000-0000-0000-000000000000.place-bottom::after {
        border-left: 8px solid transparent;
        border-right: 8px solid transparent;
        top: -6px;
        left: 50%;
        margin-left: -8px;
        border-bottom-color: #222;
        border-bottom-style: solid;
        border-bottom-width: 6px;
    }

    .t00000000-0000-0000-0000-000000000000.place-left {
        margin-left: -10px;
    }
    .t00000000-0000-0000-0000-000000000000.place-left::before {
        border-left: 8px solid transparent;
    }
    .t00000000-0000-0000-0000-000000000000.place-left::after {
        border-top: 5px solid transparent;
        border-bottom: 5px solid transparent;
        right: -6px;
        top: 50%;
        margin-top: -4px;
        border-left-color: #222;
        border-left-style: solid;
        border-left-width: 6px;
    }

    .t00000000-0000-0000-0000-000000000000.place-right {
        margin-left: 10px;
    }
    .t00000000-0000-0000-0000-000000000000.place-right::before {
        border-right: 8px solid transparent;
    }
    .t00000000-0000-0000-0000-000000000000.place-right::after {
        border-top: 5px solid transparent;
        border-bottom: 5px solid transparent;
        left: -6px;
        top: 50%;
        margin-top: -4px;
        border-right-color: #222;
        border-right-style: solid;
        border-right-width: 6px;
    }
  ",
                  }
                }
              />
              Maximum 6 pictures
            </div>
          </span>
          <div
            className="flex justify-center px-6 pt-5 pb-6 border-2 border-gray-300 border-dashed rounded-md"
          >
            <div
              className="space-y-1 text-center"
            >
              <svg
                aria-hidden="true"
                className="w-12 h-12 mx-auto text-gray-400"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 48 48"
              >
                <path
                  d="M28 8H12a4 4 0 00-4 4v20m32-12v8m0 0v8a4 4 0 01-4 4H12a4 4 0 01-4-4v-4m32-4l-3.172-3.172a4 4 0 00-5.656 0L28 28M8 32l9.172-9.172a4 4 0 015.656 0L28 28m0 0l4 4m4-24h8m-4-4v8m-12 4h.02"
                  strokeLinecap="round"
                  strokeLinejoin="round"
                  strokeWidth="2"
                />
              </svg>
              <input
                accept="image/png, image/jpg, image/jpeg, image/JPG, image/JPEG"
                className="photo-upload-width photo-upload hover:text-indigo-500"
                multiple={true}
                name="logo"
                onChange={[Function]}
                type="file"
              />
              <p
                className="text-xs text-gray-500"
              >
                PNG or JPG up to 2MB
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <label>
    <input
      className="btn btn-green form-btn w-1/2"
      onClick={[Function]}
      type="submit"
    />
  </label>
</div>
`;
